great_expectations/rule_based_profiler/profiler.py

```diff
@@ -129,10 +132,12 @@ def profile(
         self,
         *,
         expectation_suite_name: Optional[str] = None,
+        include_citation: bool = True,
     ) -> ExpectationSuite:
         """
         Args:
             :param expectation_suite_name: A name for returned Expectation suite.
+            :param include_citation: Whether or not to include Profiler's config in metadata
```

Comment:
Not sure how I feel about this docstring and the comment in L154. Can we work to make this a bit cleaner?

Reply:
@cdkini I am not seeing the major problem -- perhaps we should discuss so as not to miss anything important?  Thanks!